source/install/cloud-changelog.md

```diff
@@ -4,12 +4,43 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-12-08](#release-2022-12-08)
 - [Release 2022-12-01](#release-2022-12-01)
 - [Release 2022-11-24](#release-2022-11-24)
 - [Release 2022-11-17](#release-2022-11-17)
 - [Release 2022-11-10](#release-2022-11-10)
 - [Release 2022-10-27](#release-2022-10-27)
-- [Release 2022-10-20](#release-2022-10-20)
+
+## Release 2022-12-08
+
+### Highlights
+
+#### Message Priority and Acknowledgments (Professional plan)
+ - Added message priority labels to the Threads view.
+ - Added support for enterprise users to request acknowledgements on posts and to acknowledge posts.
+
+#### Global Drafts
+ - Added a centralized page for draft messages.
```

Comment:
@esethna - Since we have a Threads view, what do you think about us having a Drafts view too?